common/pom.xml

```diff
@@ -37,5 +37,16 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>rocketmq-remoting</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-api-mockito</artifactId>
+            <version>1.6.6</version>
```

Comment:
Have you noticed the dependency mockito in the parent pom? powermock is a good extension for mockito. I strongly recommend that you could move the dependency into parent pom and scope it in the test.

Reply:
PowerMock has been moved to the parent pom and version has been changed